config.w32

```diff
@@ -224,6 +224,52 @@ if (PHP_MONGODB != "no") {
     WARNING("mongodb libsasl support not enabled, libs not found");
   }
 
+  if (PHP_CLIENT_SIDE_ENCRYPTION != "no" && mongoc_ssl_found) {
+    mongoc_opts.MONGOC_ENABLE_CLIENT_SIDE_ENCRYPTION = 1;
+
+    PHP_MONGODB_CFLAGS += "\
+      /I" + configure_module_dirname + "/src/libmongocrypt/src \
+      /I" + configure_module_dirname + "/src/libmongocrypt/kms-message/src \
+      /I" + configure_module_dirname + "/src/libmongocrypt-compat \
+    ";
+
+    var mongocrypt_opts = {
+      MONGOCRYPT_ENABLE_TRACE: 1,
+      MONGOCRYPT_IS_POSIX: 0,
+      MONGOCRYPT_IS_WIN: 1,
+      MONGOCRYPT_ENABLE_CRYPTO: 0,
+      MONGOCRYPT_ENABLE_CRYPTO_LIBCRYPTO: 0,
+      // TODO: Support building with Secure Channel on Windows
+      MONGOCRYPT_ENABLE_CRYPTO_CNG: 0,
+      // Secure Transport does not apply to Windows
+      MONGOCRYPT_ENABLE_CRYPTO_COMMON_CRYPTO: 0
+    };
+
+    // Generated with: find src/libmongocrypt/src -name '*.c' -print0 | cut -sz -d / -f 4- | sort -dz | tr '\000' ' '
+    var PHP_MONGODB_MONGOCRYPT_SOURCES="crypto/cng.c crypto/commoncrypto.c crypto/libcrypto.c crypto/none.c mongocrypt-binary.c mongocrypt-buffer.c mongocrypt.c mongocrypt-cache.c mongocrypt-cache-collinfo.c mongocrypt-cache-key.c mongocrypt-ciphertext.c mongocrypt-crypto.c mongocrypt-ctx.c mongocrypt-ctx-datakey.c mongocrypt-ctx-decrypt.c mongocrypt-ctx-encrypt.c mongocrypt-key-broker.c mongocrypt-key.c mongocrypt-kms-ctx.c mongocrypt-log.c mongocrypt-marking.c mongocrypt-opts.c mongocrypt-status.c mongocrypt-traverse-util.c os_posix/os_mutex.c os_posix/os_once.c os_win/os_mutex.c os_win/os_once.c";
+
+    // Generated with: find src/libmongocrypt/kms-message/src -name '*.c' -print0 | cut -sz -d / -f 5- | sort -dz | tr '\000' ' '
+    var PHP_MONGODB_MONGOCRYPT_KMS_MESSAGE_SOURCES="hexlify.c kms_b64.c kms_caller_identity_request.c kms_crypto_apple.c kms_crypto_libcrypto.c kms_crypto_none.c kms_crypto_windows.c kms_decrypt_request.c kms_encrypt_request.c kms_kv_list.c kms_message.c kms_request.c kms_request_opt.c kms_request_str.c kms_response.c kms_response_parser.c sort.c";
+
+    ADD_SOURCES(configure_module_dirname + "/src/libmongocrypt/src", PHP_MONGODB_MONGOCRYPT_SOURCES, "mongodb");
+    ADD_SOURCES(configure_module_dirname + "/src/libmongocrypt/kms-message/src", PHP_MONGODB_MONGOCRYPT_KMS_MESSAGE_SOURCES, "mongodb");
```

Comment:
That's correct, I referred to that in the comment above since I didn't get to it before. I saw you added it in your PR, so this should be resolved now.